rust/src/tx_builder.rs

```diff
@@ -1917,27 +2214,21 @@ mod tests {
                 &genesis_id(),
                 0
             ),
-            &Value::new(&to_bignum(10))
+            &Value::new(&to_bignum(500))
```

Comment:
I didn't verify these differences. I'm assuming they come from refactoring out `create_tx_builder_with_fee_and_val_size()` with different values from what it was before in the tests.

Reply:
Yup, this is just updating the tests to the new min required value.